Makefile

```diff
@@ -27,4 +27,15 @@ build-hmily-admin:
 		-Djacoco.skip=true \
 		-DskipTests \
 		-Prelease \
-		clean package
\ No newline at end of file
+		clean package
+
+publish-admin-image: build-hmily-admin
+	@echo "build and push hmily admin image"
+	@docker buildx build --push \
+		--platform=linux/arm64,linux/amd64 \
+		-t ${REGISTRY}/${ADMIN_REPOSITORY}:latest \
+		-t ${REGISTRY}/${ADMIN_REPOSITORY}:${VERSION} \
+		--build-arg APP_NAME=hmily-admin-${VERSION}-admin-bin \
+		--label "commit.id=${COMMIT_ID}" \
+		-f ${HMILY_HOME}/hmily-admin-dist/docker/Dockerfile \
+		${HMILY_HOME}/hmily-admin-dist
```

Comment:
It's better to keep a blank line in each file. 

refer: https://www.dovov.com/intellij-idea-6.html